### ISSUE TITLE
External module imports are no longer always valid import alias sources

### ISSUE THREAD

**Issue by sophiajt** [microsoft/TypeScript#152]
In the following code, we now no longer assume that the result of an import is a valid source for an import alias.

``` TypeScript
//a.ts:
import im = require('b');
export module M {
    export import d = im;  // No longer valid
}

//b.ts:
function f() { }
export = f;
```

According to 10.3 in the spec an import declaration whose EntityName is a single identifier must reference an internal module (i.e. a namespace). When looking up 'im' as a namespace nothing is found ('im' is a value in the example) and thus an error is reported.
